playbooks/roles/dev-patcher/tasks/patch-repos.yml

```diff
@@ -9,29 +9,27 @@
 
 - name: Get patch {{ patchnumber }} details
   uri:
-    url: "{{ dev_patcher_baseurl }}/?q=is:open+{{ patchnumber }}&o=ALL_REVISIONS"
+    url: "{{ dev_patcher_baseurl }}/?q={{ patchnumber }}&o=ALL_REVISIONS"
```

Comment:
I think it would be better to change ALL_REVISIONS to CURRENT_REVISION here so that we don't need the "first" filter below.

Reply:
ALL_REVISIONS is there for a reason, so that we can track same revision of a patch although it changs upstream

Reply:
See also https://github.com/SUSE-Cloud/socok8s/pull/395

Reply:
Thanks for the context